src/server/controllers/item.controller.js

```diff
@@ -0,0 +1,37 @@
+const Item = require('../models/item.model');
+
+// Create new item
+const itemCreate = function(req, resp) {
+  const item = new Item({
+    name: req.body.name,
+    isOrdered: req.body.isOrdered
+  });
+
+  item.save(err => {
+    err
+      ? resp.status(400).send(err.message)
+      : resp.status(200).send('Product created successfuly!');
```

Comment:
You could return 201 Created status code (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201).
It would be good to set also Location header, and return newly created object in response (with new id).